Update search attributes related pages (#525)
temporalio/documentation: docs/system-tools/tctl.md

+-----------------------+----------+
```

There is also admin version of this command:
```bash
tctl admin cluster get-search-attributes
```
which will show you custom and system search attributes separately and also show underlying Elasticsearch index schema and system Workflow status.    

### Add new search attributes

Here is how you add a new search attribute:

```bash
tctl admin cluster add-search-attributes --name ProductId --type Keyword
```

The possible values for `--type` are:

- String
- Keyword
- Int
- Double
- Bool
- Datetime

> ::note Due to Elasticsearch limitations you can only add new custom search attributes but not rename or remove existing ones.

Reviewer: ```suggestion
:::note 

Due to Elasticsearch limitations you can only add new custom search attributes but not rename or remove existing ones.

:::
```

### Remove search attributes

Here is how you remove an existing search attribute:

```bash
tctl admin cluster remove-search-attributes --name ProductId
```

> ::note Due to Elasticsearch limitations this command will only remove search attribute from cluster metadata (Workflows won't be able to use them)

Reviewer: ```suggestion
:::note 

Due to Elasticsearch limitations, this command removes search attributes only from cluster metadata (Workflows won't be able to use them)
```

> but not from Elasticsearch index schema. You need to modify Elasticsearch index manually (it will require reindex in the most cases).

Reviewer: ```suggestion
but not from the Elasticsearch index schema. You need to modify the Elasticsearch index manually; in most cases, this requires reindexing.

:::
```
